notebooks/source/index.rst

```diff
@@ -16,6 +16,7 @@ Welcome to NumPyro Tutorials!
    ordinal_regression
    bayesian_hierarchical_linear_regression
    discrete_imputation
+   model_rendering
```

Comment:
Could you also add the tutorial to the end of the section "Introductory Tutorials" in this [index file](https://github.com/pyro-ppl/numpyro/blob/master/docs/source/index.rst)?